.golangci.yml

```diff
@@ -1,11 +1,21 @@
+linters:
+  enable-all: true
+  disable:
+    - dogsled # The rationale for disabling dogsled is that all the cases it flags have been false positives in test code that intentionally ignores many return values because they test only one thing.
+    - dupl # I think that one creates too many false positives as well. Our tests are kind of repetitive, which is intentional. They are supposed to be simple, straightforward scripts.
+    # - exhaustive        TODO: enable after next release; current release at time of writing is v1.27
+    - gochecknoglobals # We are currently using some globals. It's a lot more forgivable in a CLI tool than in a library.
+    - gochecknoinits # I think the CLI framework we use requires them.
+    - goconst # Our tests contain a ton of hard-coded test strings, for example branch names.
+    - goerr113 # Expose errors properly, once/if approaching library use, as opposed to current cli-only use
```

Comment:
We don't want to disable `goerr113` completely. It has many useful error checks, for example making sure we compare errors with `errors.Is`. We only want to disable the `do not define dynamic errors` check because that one applies to libraries only. That's what I did in the `exclude-rules` section. Can you please restore that section and remove `goerr113` from here?